go.mod

```diff
@@ -3,14 +3,6 @@ module github.com/containerd/nri
 go 1.14
 
 require (
-	// when updating containerd, adjust the replace rules accordingly
-	github.com/containerd/containerd v1.5.0-beta.3
+	github.com/opencontainers/runtime-spec v1.0.3-0.20200929063507-e6143ca7d51d
```

Comment:
This is the version that was used by containerd v1.5.0-beta.3; https://github.com/containerd/containerd/blob/v1.5.0-beta.3/go.mod#L45C2-L45C77

It's 12 commits ahead of runtime-spec v1.0.2; https://github.com/opencontainers/runtime-spec/compare/v1.0.2...e6143ca7d51d

We _could_ consider using a tagged release here 🤔

Reply:
nod to using tagged if possible